noxfile.py

```diff
@@ -0,0 +1,169 @@
+import os
+from contextlib import contextmanager
+from inspect import cleandoc
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+import nox
+from pyodbc import connect
+
+PROJECT_ROOT = Path(__file__).parent
+
+# default actions to be run if nothing is explicitly specified with the -s option
+nox.options.sessions = ["verify"]
+
+
+class Settings:
+    ITDE = PROJECT_ROOT / ".." / "integration-test-docker-environment"
```

Comment:
if you want to do it as script, I suggest you clone the ITDE to a temporary directory

Reply:
Currently this is how on the local machine and the github action the  projects are setup. That said I know having that hard coded is not ideal but I would rather leave it like that and push `ITDE` so that it can be added as package dependency in
the `requirements_dev.txt`.

Reply:
Hm, for local tests it might be interesting to run the tests with an already started DB, because the start of ITDE takes some time. This is important in the development phase when you run tests many times. For that, it's only important to implement a proper cleanup. Is the idea to run locally then only with pytest?